week1/homework/src/server.js

```diff
@@ -5,16 +5,44 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+function handleRequest(response, state) {
```

Comment:
🤔 Is `handleRequest `a descriptive name for this function?

Reply:
I Thought it was since it handles the request which comes from the client. :) So what do u think it could be a better name?

Reply:
You are right. Technically the function is handling a request. However, for a node application almost every function is related to handling requests :). The name should be more specific. 

Naming things is perhaps the hardest thing of all in programing. Here is a [nice post](https://dzone.com/articles/naming-conventions-from-uncle-bobs-clean-code-phil) based on a great book from Robert Martin aka Uncle Bob if you are interested.